scripts/bump-version.sh

```diff
@@ -10,7 +16,7 @@ TARGET="${2}"
 echo "Current version: $VERSION"
 echo "Bumping version: $TARGET"
 
-sed -i '' -e "1,/^version/ s/^version.*/version = \"$TARGET\"/" Cargo.toml
+perl -pi -e "s/^version = .*\$/version = \"$TARGET\"/" Cargo.toml
```

Comment:
Note that this will now replace all occurrences and not just the first. Since this matches the start of the line, however, this should be fine.

Reply:
PERL!

![invincible](https://media.giphy.com/media/4QEGvof07T93uq1PwH/giphy-downsized.gif)